### ISSUE TITLE
Update context processor paths

### ISSUE THREAD

**Issue by melinath** [dancerfly/django-brambling#779]
As part of the django 1.8 upgrade, [context processor paths are changing](https://docs.djangoproject.com/en/1.9/releases/1.8/#django-core-context-processors). We should update them once #771 is merged (so it'll be easy to do in both test & prod settings at once.)

**Comment by melinath** [comment]
This is no longer relevant
